godotcord_store_manager.h

```diff
@@ -2,49 +2,122 @@
 #define GODOTCORD_STORE_MANAGER_H
 
 #include "core/object.h"
+#include "core/reference.h"
+#include "godotcord_utils.h"
+
+class GodotcordSKUPrice : public Reference {

```

Comment:
I think its too many classes in this file. Maybe you could move the "data" class into another header file.

Reply:
Oh, I'd move it to different header file instead.